src/content/reference/manifest.mdx

```diff
@@ -610,6 +610,34 @@ volumes:
 
 Sets the working directory of your development container.
 
+### forward ([string], optional)
+
+Using a global forward, Okteto handles port collision if two `okteto up` sessions are running at the same time. 
```

Comment:
```suggestion
When declaring a global forward, Okteto will automatically handle port collision when two or more `okteto up` sessions are running simultaneously. If the`okteto` detects that the port is already in use, and said port is defined as global forward,  `okteto up` will ignore the port collision and continue the `up` sequence. If the port is later available, `okteto` will automatically connect to it without interrupting the session.   This is disabled by default. 
```

Reply:
@pchico83 @jmacelroy @AdrianPedriza what do you think about my explanation here? This one is tricky, i want to make sure is clear for everyone that reads it.

Reply:
much more accurate but I'm not quite clear what the last sentence (`This is disabled by default`) refers to.